pkg/daemon/ceph/agent/agent.go

```diff
@@ -76,12 +77,21 @@ func (a *Agent) Run() error {
 	}
 
 	flexDriverVendors := []string{flexvolume.FlexvolumeVendor, flexvolume.FlexvolumeVendorLegacy}
-	for _, vendor := range flexDriverVendors {
+	for i, vendor := range flexDriverVendors {
+		if i > 0 {
+			// Wait before the next driver is registered. In 1.11 and newer there is a timing issue if flex drivers are registered too quickly.
+			// See https://github.com/rook/rook/issues/1501 and https://github.com/kubernetes/kubernetes/issues/60694
+			time.Sleep(500 * time.Millisecond)
```

Comment:
has there been much testing runs to see if this is making a difference, or do we hope that 500ms is a good value to use?

Reply:
oh nevermind, i see this now:
> With this change, note that after eight jenkins builds of this PR, we have not hit the flex issue yet.